phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/PhoenixIndexImportDirectReducer.java

```diff
@@ -49,8 +50,42 @@
     private static final Logger LOGGER =
             LoggerFactory.getLogger(PhoenixIndexImportDirectReducer.class);
 
+    private static final String SPLIT_MERGE_ERROR_STR_FMT =
```

Comment:
This solution tackles Split-Only case. With concurrent merges, this is not a guarantee that region counts would differ.